cdap-api/src/main/java/io/cdap/cdap/api/plugin/PluginConfigurer.java

```diff
@@ -95,4 +101,33 @@
   @Nullable
   <T> Class<T> usePluginClass(String pluginType, String pluginName,
                               String pluginId, PluginProperties properties, PluginSelector selector);
+
+  /**
+   * Evaluates lookup macros and the 'secure' macro function using provided macro evaluator.
```

Comment:
nit: Evaluates lookup macros and macro functions using... 

The secure macro function isn't a platform level thing so it shouldn't be mentioned in the javadoc.